src/test/java/org/gridsuite/modification/server/modifications/OperatingStatusModificationLockoutLineTest.java

```diff
@@ -79,35 +80,53 @@ protected void assertAfterNetworkModificationDeletion() {
         TestUtils.assertOperatingStatus(getNetwork(), TARGET_LINE_ID, OTHER_BRANCH_STATUS);
     }
 
+    @Test
+    public void testCreate() throws Exception {
+        OperatingStatusModificationInfos modificationInfos = (OperatingStatusModificationInfos) buildModification();
+        //test lockout line modification with non-fictional switch
+        getNetwork().getSubstation("s1").newVoltageLevel().setId("v1f").setFictitious(true)
+                .setTopologyKind(TopologyKind.NODE_BREAKER).setNominalV(380.).add();
+        getNetwork().getSubstation("s2").newVoltageLevel().setId("v3f").setFictitious(true)
+                .setTopologyKind(TopologyKind.NODE_BREAKER).setNominalV(380.).add();
+
+        Line line = getNetwork().newLine()
+                .setId("nonFictionalDisconnect")
+                .setVoltageLevel1("v1f")
+                .setVoltageLevel2("v3f")
+                .setNode1(0)
+                .setNode2(0)
+                .setX(12)
+                .setR(7)
+                .add();
+        assertNotNull(line);
+
+        line.getTerminal("v1f").disconnect(SwitchPredicates.IS_NONFICTIONAL);
```

Comment:
Don't understand !
Need to create a fictional switch and test disconnect
And another tests each with a type of switch ?